blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/UserImpl.java

```diff
@@ -6,6 +6,8 @@
 import hudson.model.User;
 import hudson.tasks.Mailer;
 import hudson.tasks.UserAvatarResolver;
+import hudson.util.HttpResponses;
```

Comment:
@kzantow it extends from `org.kohsuke.stapler.HttpResponses` so it could return this type as well in method signature. 

You may like to add method signature to BlueUser with stapler one (as we do not add jenkins model classes in blueocean-rest module. This could come in later PR though.